src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -226,11 +226,15 @@ function ReportPreview({
     const shouldShowSubtitle = !isScanning && (shouldShowSingleRequestMerchantOrDescription || numberOfRequests > 1);
 
     const {supportText} = useMemo(() => {
-        if (formattedMerchant) {
-            return {supportText: formattedMerchant};
+        if (formattedMerchant && formattedMerchant !== CONST.TRANSACTION.DEFAULT_MERCHANT) {
+            return {isSupportTextHtml: false, supportText: formattedMerchant};
         }
         if (formattedDescription ?? moneyRequestComment) {
-            return {supportText: formattedDescription ?? moneyRequestComment};
+            const parsedSubtitle = new ExpensiMark().replace(formattedDescription ?? moneyRequestComment);
```

Comment:
It seems this change is outdated

Reply:
@DylanDylann i updated, please check again